packages/website/pages/docs/concepts/bridging.mdx

```diff
@@ -64,11 +64,11 @@ Let's walk through an example:
 2. Next, we call `eth_getProof`, which will give us a proof that we did indeed send a message on the source chain.
 3. Finally, we call `isSignalReceived` on the destination chain's SignalService which essentially just verifies the merkle proof. `isSignalReceived` will look up the block hash you are asserting you had stored a message on the source chain (where you originally sent the message), and with the sibling hashes inside the merkle proof it will rebuild the merkle root, which verifies the signal was included in that merkle root—meaning it was sent.
 
-And boom! We have sent a cross-chain message. Confusing? Well leave some feedback please if it is, so we can clear it up.
+And boom! We have sent a cross-chain message. Confusing? Well, leave some feedback please if it is, so we can clear it up but a simple dApp was built during one of our workshops to demonstrate the fundamentals. You can find it [here](https://github.com/adaki2004/MessageService).
```

Comment:
I cloned your app into Taiko account, could you use this one instead: https://github.com/taikoxyz/MessageServiceShowCaseApp